caluma/caluma_core/filters.py

```diff
@@ -425,10 +427,19 @@ def filter(self, qs, value):
                     ),
                 )
                 lookup = {f"field_val__{lookup_expr}": expr["value"]}
+            elif lookup_expr == JSONLookupMode.INTERSECTS:
+                lookup = {}
+                exprs = [
+                    Q(**{f"{self.field_name}__{expr['key']}__contains": val})
```

Comment:
Did you check whether we need the transformation above (L422-L429) or not? Maybe add a test using strings as values instead of integers.

Reply:
Nice! In that case, could you shortly check if we even need the transform above anymore?

Reply:
Tried commenting out the transform and it broke the contains, seems still necessary